chainerx_cc/chainerx/cuda/cudnn.cc

```diff
@@ -96,6 +98,11 @@ CudnnTensorDescriptor::~CudnnTensorDescriptor() {
     }
 }
 
+CudnnTensorDescriptor::CudnnTensorDescriptor(CudnnTensorDescriptor&& other) {
+    std::swap(desc_, other.desc_);
+    other.desc_ = nullptr;
```

Comment:
How about `desc_{other.desc_}` in the initializer list?

Reply:
Ah that's simpler. Thanks, fixed!